kayenta-atlas/src/main/java/com/netflix/kayenta/atlas/metrics/AtlasMetricsService.java

```diff
@@ -146,12 +148,10 @@ private AtlasNamedAccountCredentials getCredentials(String accountName) {
     long start = registry.clock().monotonicTime();
     List <AtlasResults> atlasResultsList;
     try {
-      atlasResultsList = atlasRemoteService.fetch(decoratedQuery,
-                                                  atlasCanaryScope.getStart().toEpochMilli(),
-                                                  atlasCanaryScope.getEnd().toEpochMilli(),
-                                                  isoStep,
-                                                  credentials.getFetchId(),
-                                                  UUID.randomUUID() + "");
+      atlasResultsList = retry.retry(() -> atlasRemoteService.fetch(decoratedQuery,
+                                                                    atlasCanaryScope.getStart().toEpochMilli(),
+                                                                    atlasCanaryScope.getEnd().toEpochMilli(), isoStep, credentials.getFetchId(), UUID.randomUUID() + ""),
+      20, 1000);
```

Comment:
The indentation looks a bit weird here. retry() is passed 3 arguments, right?

Reply:
A future reader would have a hard time knowing where the `20` and `1000` came from. Maybe we could use constants for these and `TimeUnit` where it makes sense.